### PR TITLE
1490 - Add NPC-specifc prefabs into class_prefab.c inside Playerclass

### LINKED ISSUE #1490: Add NPC-specifc prefabs into class_prefab.c inside Playerclass
Jonathan, Katie, and I are adding in friendly NPC's into the playerclass module in class_prefabs.c. Currently, all the playerclass prefabs are very combat and skill heavy, and there aren't many supportive classes. Therefore, we are going to add some classes into the code with the stats of these new classes, alongside a short and long description. Although there is room for refactoring, we will primarily be focused on just adding these new classes in the class_prefab.c file inside Playerclass.

### PR DESCRIPTION
**Pull request by jroven**
Vincent, Katie, and I have added in friendly NPC's into the playerclass module in class_prefabs.c. Before, all the playerclass prefabs were very combat and skill heavy, and there weren't many supportive classes. Therefore, we added some classes into the code with the stats of these new classes, alongside a short and long description. Katie added lord and healer, Vincent added alchemist and chef, and I added fisherman and shopkeeper. We made a space change in wdl/test_class.c to match monk, and changed some spacing/if else statements to have the code compile and past the tests currently written.

### CODE REVIEW

File: include/playerclass/class_prefabs.h [hunk 1]
```diff
@@ -39,11 +39,6 @@ class_t* class_prefab_new(game_t* game, char* class_name);
  * Returns:
  *  - status code (FAILURE if the given class or skill is NULL, SUCCESS otherwise.)
  *  - the structs in the class now contain skills.
- *
```
Comment (vanolson) [inline include/playerclass/class_prefabs.h:42]: Was wondering about this deletion - we're still waiting on the developed skill effects, right? So might want to keep that part of the comment.

**Review by vanolson — APPROVED** [review]
Looks good! I was wondering about the comments deleted at line 42 of class_prefabs.h, before the `class_prefab_add_skills()` function—it might be useful to keep parts of what was deleted, since we are still waiting for the skill effects to be developed further, I believe. But that's a small change, just something I was unsure about. Besides that, this looks great! I really like the added NPC classes, and I think the variation in stats between them is good.

File: src/playerclass/src/class_prefabs.c [hunk 2]
```diff
@@ -252,6 +277,109 @@ class_t* class_prefab_new(game_t* game, char* class_name) {
...
+     * 10 Max Health
+     * 20 Speed
+     * 5 Physical Defense
+     * 5 Physical Attack
+     * 10 Ranged Attack
+     * 20 Magic Defense
+     * 25 Magic Attack
+     * 35 Max Mana */ 
+    else if (!strncmp(temp_name, "alchemist", MAX_NAME_LEN)) {
+        short_desc = "a skillfull alchemist.";
```
Comment (lilyehsani) [inline src/playerclass/src/class_prefabs.c]: Capitalize sentence

**Review by lilyehsani — CHANGES_REQUESTED** [review]
Looking good! 1 minor comment, and Van also made a good point about the comment in class_prefabs.h. Also resolve the merge conflicts.

File: src/playerclass/src/class_prefabs.c (hunk 2 above)
Comment (katie-gao) [reply src/playerclass/src/class_prefabs.c]: Made the change!

**Review by lilyehsani — APPROVED** [review]
LGTM! Approved.

**Comment by lilyehsani** [comment]
Issue Score: Excellent

Comments:
Great work! Like issues, PRs should have closing statements.

**Pull request merged**